test/services/stake_pools/calculate_average_stats_test.rb

```diff
@@ -80,10 +80,10 @@ class CalculateAverageStatsTest < ActiveSupport::TestCase
 
       assert_equal 3, stake_pool.average_uptime
       assert_equal 10, stake_pool.average_lifetime
-      assert_equal 0, stake_pool.average_delinquent
       assert_equal 2.55, stake_pool.average_skipped_slots
       assert_equal 7, stake_pool.average_score
-      assert_equal 8.15, stake_pool.average_validators_commission
+      assert_equal 6.1, stake_pool.average_validators_commission
```

Comment:
why is this value changed?

Reply:
`active_stake` value has been changed and it has an impact on average_commission calculation.